scripts/aspnet.benchmarks.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=ASP.NET Benchmarks Server
+
+# Start after network has at least a bound IP address; stop before network is stopped
+After=network.target
+After=network-online.target
```

Comment:
Discussed offline and I'll try a few reboots without `After=network-online.target` to confirm we don't need it. (If there's something ASP.NET Core-specific here, it's not currently documented.)